zenodo/modules/records/serializers/schemas/datacite.py

```diff
@@ -539,7 +534,14 @@ def get_fundingreferences(self, obj):
             funder_identifier = g.get('funder', {}).get('doi')
             award_program = g.get('program')
             if award_program:
-                for funder_id, regexes in self.FUNDER_PROGRAM_MATCHES.items():
+                funder_program_matches = current_app.config.get(
+                    'ZENODO_FUNDER_PROGRAM_MATCHES', {
+                        '10.13039/100010661': ('^H2020$', '^Horizon 2020',),
+                        '10.13039/100011102': ('^FP7$',),
+                        '10.13039/100018693': ('^HE$', '^Horizon Europe',),
+                    }
+                )
```

Comment:
```suggestion
                    'ZENODO_FUNDER_PROGRAM_MATCHES', {})
```

We can leave it empty by default, since there's always going to be something from default config.